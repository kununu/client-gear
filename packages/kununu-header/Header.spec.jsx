import React from 'react';
import {render} from 'enzyme'; // eslint-disable-line import/no-extraneous-dependencies

import {Header, HeaderNav, HeaderNavItem} from './index';


test('Renders Header without crashing', () => {
  const component = render(
    <Header
      title="Volle Transparenz am Arbeitsmarkt"
      logoLink={<a href="/test">test</a>}
    >
      <HeaderNav>
        <HeaderNavItem>
          <a href="/test">
            <span className="hidden-xs">
              <i
                className="fa fa-search hidden-xs"
                aria-hidden="true"
              />
              &nbsp; Suchen
            </span>
            <i
              className="fa fa-search visible-xs"
              aria-hidden="true"
            />
          </a>
        </HeaderNavItem>
        <HeaderNavItem>
          <a
            className=""
            href="/"
          >
            <span className="hidden-xs">Mein kununu </span>
            <i className="fa fa-user visible-xs" />
          </a>
        </HeaderNavItem>
        <HeaderNavItem>
          <a
            href="/"
            className="btn btn-dd-sm btn-primary"
          >
            Firma bewerten
          </a>
        </HeaderNavItem>
      </HeaderNav>
    </Header>,
  );

  expect(component).toMatchSnapshot();
});
