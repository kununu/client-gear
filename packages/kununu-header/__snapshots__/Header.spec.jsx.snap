// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Header without crashing 1`] = `
<header
  class="header fixed"
  role="banner"
>
  <div
    class="container-fluid"
  >
    <div
      class="flex"
    >
      <div
        class="pullLeft"
      >
        <a
          href="/test"
        >
          <img
            alt="Volle Transparenz am Arbeitsmarkt"
            class="logoDesktop"
            src="https://assets.kununu.com/images/footer/logo-desktop.svg"
          />
          <img
            alt="Volle Transparenz am Arbeitsmarkt"
            aria-hidden="true"
            class="logoMobile"
            rel="presentation"
            src="https://assets.kununu.com/images/footer/logo-mobile.svg"
          />
        </a>
        <span
          class="title"
        >
          Volle Transparenz am Arbeitsmarkt
        </span>
      </div>
      <div
        class="pullRight"
      >
        <nav
          class="headerNav"
        >
          <ul>
            <li
              class="headerNavItem"
            >
              <a
                href="/test"
              >
                <span
                  class="hidden-xs"
                >
                  <i
                    aria-hidden="true"
                    class="fa fa-search hidden-xs"
                  />
                  Â  Suchen
                </span>
                <i
                  aria-hidden="true"
                  class="fa fa-search visible-xs"
                />
              </a>
            </li>
            <li
              class="headerNavItem"
            >
              <a
                class=""
                href="/"
              >
                <span
                  class="hidden-xs"
                >
                  Mein kununu 
                </span>
                <i
                  class="fa fa-user visible-xs"
                />
              </a>
            </li>
            <li
              class="headerNavItem"
            >
              <a
                class="btn btn-dd-sm btn-primary"
                href="/"
              >
                Firma bewerten
              </a>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </div>
</header>
`;
